### PR TITLE
README improvements

### PR DESCRIPTION
**Pull request by wevtimoteo**
- [x] Add `ubuntu` install instructions
- [x] Fix typos

### CODE REVIEW

**Comment by wevtimoteo** [comment]
Now its done ;)

Feel free to merge it

**Comment by wevtimoteo** [comment]
wait, one more typo found

**Comment by wevtimoteo** [comment]
Done

File: README.md [hunk 1]
```diff
@@ -1,23 +1,26 @@
-#Emacs Fast Start
+Emacs Fast Start
+----------------
 
-This is a repo to help users to start use Emacs. In this code I choose the essencial packages and configurations to have a happy start with this amazing editor.
+This repo is proposed to help users to start using `emacs` editor. In this code I choose the essential packages and configurations to have a happy start with this amazing editor.
```
Comment (squiter) [inline README.md]: We need to use `Emacs` instead `emacs`.

File: README.md [hunk 2]
```diff
@@ -43,6 +47,23 @@ I'm a OSX user and I recommend you to use one of these builds for Emacs:
 * [Emacs Mac Port](https://github.com/railwaycat/homebrew-emacsmacport/releases);
 * To know all diferences for the builds I recommend you to read this [Stack Overflow answer](http://emacs.stackexchange.com/questions/271/what-is-the-difference-between-aquamacs-and-other-mac-versions-of-emacs)
 
+For Ubuntu you can:
+
+```
+cd /tmp/
+wget http://alpha.gnu.org/gnu/emacs/pretest/emacs-25.0.93.tar.xz
+tar -xvf $(EMACS).tar.xz
```
Comment (squiter) [inline README.md]: Use `emacs-25.0.93` here, or use `$(EMACS)` in the other places.

Comment (wevtimoteo) [inline README.md]: Nice catch :baseball:

File: README.md (hunk 1 above)
Comment (wevtimoteo) [inline README.md]: Done in ed7846c

File: README.md (hunk 2 above)
Comment (wevtimoteo) [inline README.md]: Done in ed7846c

File: README.md (hunk 1 above)
Comment (wevtimoteo) [inline README.md]: I see, didn't know about it.

Done in cfe0f04

**Comment by squiter** [comment]
:+1: :heart: :+1:

**Pull request merged**